uwblueprint/pregnancy-centre
# Auth sign up

This PR addresses this ticket: https://github.com/uwblueprint/pregnancy-centre/issues/15

What's new
- SignUp modal which creates a user through Firebase
- Password requirements popover which checks that password meets password requirements in real time
- Styling for the modal

To test this PR
- checkout this branch and go to /signup
- create a user and go to https://console.firebase.google.com/u/1/project/bp-pregnancy-centre/authentication/users to ensure that user has been created
- play around with password to check that password requirements popover working

Please ignore App.tsx, CommonModal.tsx, ConfirmationModal.tsx, and Modal.scss
